wave_vision/include/wave/vision/matcher/flann_matcher.hpp

```diff
@@ -179,6 +179,20 @@ struct FLANNMatcherParams {
  */
 class FLANNMatcher : public DescriptorMatcher {
  public:
+    // Diagnostics information
+    /** The number of matches prior to any filtering or outlier removal. */
+    size_t num_raw_matches = 0u;
+
+    /** The number of matches after being filtered by the distance/ratio test.
+     *  This is the input to the outlierRejection method.
+     */
+    size_t num_filtered_matches = 0u;
+
+    /** The size of matches remaining after outlier removal via RANSAC or a
+     *  similar method.
+     */
+    size_t num_good_matches = 0u;
```

Comment:
Put these into the DescriptorMatcher class?

Reply:
fixed, thank you.